core/http_api.php

```diff
@@ -185,6 +185,12 @@ function http_csp_value() {
 
 	$t_csp_value = '';
 
+	# frame-ancestors can't have 'none' together with other values.
+	if( count( $g_csp['frame-ancestors'] ) > 1 ) {
```

Comment:
That's not strictly necessary, considering that we always set `'none'` in http_security_headers().

But I guess it would not hurt to go for belt & braces and add this as a safety net.